src/GitHub.VisualStudio.UI/Views/Dialog/RepositoryCreationView.xaml

```diff
@@ -60,19 +62,18 @@
             FocusVisualStyle="{x:Null}">
 
         <Grid.Resources>
-          <Style TargetType="{x:Type TextBlock}">
+                    <Style TargetType="{x:Type TextBlock}" BasedOn="{StaticResource {x:Static ui:VsResourceKeys.ThemedDialogTextBoxStyleKey}}">
```

Comment:
The target type here is for `TextBlock` and you seem to be applying a `TextBox` style to it.